letsencrypt-win-simple/Plugin/OverviewPlugin.cs

```diff
@@ -0,0 +1,154 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using Serilog;
+using Microsoft.Web.Administration;
+
+namespace LetsEncrypt.ACME.Simple
+{
+    public class OverviewPlugin : Plugin
+    {
+        private const string ClientName = "letsencrypt-win-simple";
```

Comment:
This should not be hard coded in here. However there isn't currently a way in the main Program to get this value. So it needs to be changed to Public in the program and then you should use Program.ClientName wherever you need to use it.